### PR TITLE
Refs #3593: Fix documentation of quasi-static machine examples

### PR DESCRIPTION
**Pull request by christiankral**
I checked the quasi static induction machine examples and updated the documentation to match the actual variables.

### CODE REVIEW

**Comment by christiankral** [comment]
The following models use the instance name `iSensorQS` instead of `currentQuasiRMSSensorQS` which makes the example collection a little inconsistent 
- `Modelica.Magnetic.QuasiStatic.FundamentalWave.Examples.BasicMachines.InductionMachines.IMC_Characteristics` 
- `Modelica.Magnetic.QuasiStatic.FundamentalWave.Examples.BasicMachines.InductionMachines.IMS_Characteristics`

@tobolar Are we allowed to rename a sensor in the master branch? If yes, I would unify the appearance of the sensor names.

**Comment by christiankral** [comment]
There is one more issue with a synchronous machine example which I am going to fix.

File: Modelica/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Conveyor.mo [hunk 1]
```diff
@@ -201,9 +201,9 @@ The mechanical load is a constant torque like a conveyor (with regularization ar
 </p>
 <p>Simulate for 20 seconds and plot (versus time):</p>
 <ul>
-<li>currentQuasiRMSSensor|currentQuasiRMSSensorQS.I: stator current RMS</li>
-<li>imc|imcQS.wMechanical: machine speed</li>
-<li>imc|imcQS.tauElectrical: machine torque</li>
+<li><code>currentQuasiRMSSensor|currentQuasiRMSSensorQS.I</code>: (equivalent) stator current RMS</li>
+<li><code>imc|imcQS.wMechanical</code>: machine speed</li>
+<li><code>imc|imcQS.tauElectrical</code>: machine torque</li>
```
Comment (tobolar) [inline Modelica/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Conveyor.mo:206]: Shouldn't the options be enclosed by parentheses? Thus e.g. `<code>(currentQuasiRMSSensor|currentQuasiRMSSensorQS).I</code>`.

**Comment by tobolar** [comment]
> Are we allowed to rename a sensor in the master branch? If yes, I would unify the appearance of the sensor names.

Do you mean to rename a sensor element in that particular example model?

**Comment by christiankral** [comment]
> > Are we allowed to rename a sensor in the master branch? If yes, I would unify the appearance of the sensor names.
> 
> Do you mean to rename a sensor element in that particular example model?

Yes, this is what I mean

**Comment by tobolar** [comment]
IMO this is no problem when only an example is concerned. I remember there has been some comment on it by (I guess) @dietmarw but I can't find it anymore.

**Comment by dietmarw** [comment]
First of all you need to determine what  is meant by "master branch". The question is what next version the master branch is aiming at. Backward compatible, i.e., 4.1.0, or non-backward compatible, i.e., 5.0.0. As far as I know this has not yet been decided within MAP-LIB. In the latter case there is no restrictions on renames as long as conversion scripts are given. In the former case we tended to allow renames within examples even if they break backward compatibility since the argument is that users should not let their usermodels extend from examples (copying is fine but not extend).

**Review by AHaumer — APPROVED** [review]

**Comment by christiankral** [comment]
OK, I propose to separate the issues which became mixed up in this PR. 

- Fix documentation: this is covered by this PR
- Change spelling (currently, e.g., `smpm|smpmQS.tauElectrical`) of [multiple similar variables](https://github.com/modelica/ModelicaStandardLibrary/pull/3620#discussion_r484284023) as suggested by @tobolar: I propose to create a different ticket and PR, respectively, as this issue affects some more models of the MSL
- Unify naming of sensor: Optionally create a new PR as this issue is not directly related with the documentation; it may also require some more discussion if we are fine with renaming instance names of MSL examples in the future

So if there are not objections with this approach, I am asking @tobolar and @dietmarw to re-evaluate this PR so that we can proceed.

**Review by dietmarw — APPROVED** [review]
Looks good to me. I don't think parentheses would be strictly necessary in case of the choices but would not harm either. I leave that to the LOs to decide.

**Comment by dietmarw** [comment]
I've rebased the PR on current master in order to get the CI checks working.

**Pull request merged**